src/Composer/StudioPlugin.php

```diff
@@ -65,45 +74,80 @@ public function symlinkStudioPackages()
         // Get local repository which contains all installed packages
         $installed = $this->composer->getRepositoryManager()->getLocalRepository();
 
-        foreach ($intersection as $package) {
+        foreach ($this->getManagedPackages() as $package) {
             $original = $installed->findPackage($package->getName(), '*');
+            $originalPackage = $original instanceof AliasPackage ? $original->getAliasOf() : $original;
 
-            $installationManager->getInstaller($original->getType())
-                ->uninstall($installed, $original);
+            // Change the source type to path, to prevent 'The package has modified files'
+            if ($originalPackage instanceof CompletePackage) {
+                $originalPackage->setInstallationSource('dist');
+                $originalPackage->setDistType('path');
+            }
 
-            $installationManager->getInstaller($package->getType())
-                ->install($studioRepo, $package);
+            $installationManager->getInstaller($original->getType())->uninstall($installed, $original);
+            $installationManager->getInstaller($package->getType())->install($LocalPackagesRepo, $package);
```

Comment:
@ryzr I am getting issues since this commit. `$LocalPackagesRepo` is not defined. I have PR'ed to your fork https://github.com/ryzr/studio/pull/1